clients.go

```diff
@@ -19,6 +20,7 @@ type ClientsConfig struct {
 type Clients struct {
 	logger micrologger.Logger
 
+	dynClient  dynamic.Interface
```

Comment:
As opposed to other packages `dynamic.NewForConfig` returns interface.

I don't know if `dynClient` is the best name. I follows the patter of three letters following `Client` so I went this rabbit hole :D